ExeterTeam25/group-software-project
# Like responses

Can now like responses and each like is an additional point for the creator.

To test:

Login as a user and like a response to a challenge.
Then log in as the user who created the response you liked and make sure the points went up by 1.
